src/server/serializer_test.ts

```diff
@@ -22,6 +22,17 @@ Deno.test("serializer - primitives & plain objects", async (t) => {
   assertEquals(deserialized, data);
 });
 
+Deno.test("serializer - bigint", async (t) => {
+  const data = { a: 999999999999999999n };
+  const res = serialize(data);
+  console.log(res);
```

Comment:
Super tiny nit: This can probably be dropped. Most likely a leftover from a debugging session :)

Reply:
Oh lol, forgot to take that out. Nice catch!